app.js

```diff
@@ -29,63 +29,80 @@ var storeHoursTotal = [];
 
 var stores = [firstAndPike, seaTacAirport, seattleCenter, capitolHill, alki];
 
+//create table element HEADER FUNCTION BEGIN
 var tableEl = document.createElement('table');
-
+//create header elemenet
 var headerRowEl = document.createElement('tr');
 tableEl.appendChild(headerRowEl);
-
+//create blank element for hours to begin
 var blankEl = document.createElement('th');
 headerRowEl.appendChild(blankEl);
-
+//header row hours
 for (var i = 0; i < storeHours.length; i++) {
   var headerHourEl = document.createElement('td');
   headerHourEl.textContent = storeHours[i];
   headerRowEl.appendChild(headerHourEl);
 }
-
+//header table data for daily totals
 var headerDailyTotalEl = document.createElement('td');
 headerDailyTotalEl.textContent = 'Daily Total';
 headerRowEl.appendChild(headerDailyTotalEl);
-
-//CookieStore.prototype.createRow = function(){
-  //this.getAvgCookieCount();
+//HEADER FUNCTION END
+//for loop for stores average cookies MAIN CONTENT FUNCTION
 for (var i = 0; i < stores.length; i++) {
   var currentStore = stores[i];
   currentStore.getAvgCookieCount();
-
+//row element for store names
   var rowEl = document.createElement('tr');
   tableEl.appendChild(rowEl);
-
+//store name element appending to the row
   var nameEl = document.createElement('th');
   nameEl.textContent = currentStore.name;
   rowEl.appendChild(nameEl);
-
+//for loop for get the store hours avg for each store
   for (var j = 0; j < currentStore.hoursOpen.length; j++) {
     var storeHourEl = document.createElement('td');
     storeHourEl.textContent = currentStore.hoursOpen[j];
     rowEl.appendChild(storeHourEl);
   }
+  //daily totals table data element
   var dailyTotalEl = document.createElement('td');
   dailyTotalEl.textContent = currentStore.dailyTotal;
   rowEl.appendChild(dailyTotalEl);
 }
-
+//MAIN CONTENT FUNCTION END
+//creates total header element on the left column
+//FOOTER FUNCTION
+var totalsRowEl = document.createElement('tr');
```

Comment:
This isn't a function yet either! But it'll be easier for you in the long run if you make it one.